src/gt4py/next/backend.py

```diff
@@ -134,6 +135,16 @@ def step_order(self, inp: INPUT_PAIR) -> list[str]:
 
 DEFAULT_TRANSFORMS: Transforms = Transforms()
 
+# FIXME[#1582](havogt): remove after refactoring to GTIR
+_foast_to_itir_step = foast_to_itir.adapted_foast_to_itir_factory(cached=True)
```

Comment:
is there any particular reason why this is not inlined?

Reply:
Yes, in order to share the cache.

Reply:
Makes sense! Please add a comment, to avoid accidentally inlining...